src/BizHawk.Client.EmuHawk/MainForm.cs

```diff
@@ -768,6 +768,17 @@ private set
 			}
 		}
 
+		private bool _block_frame_advance;

```

Comment:
There's no reason for this backing value.  You can just have public BlockFrameAdvance { get; set; }